pkg/projects/descriptions.json

```diff
@@ -5,6 +5,11 @@
         "CommonTypes": [ ]
     },
     {
+       "Name": "NuGet3MinVersion",
```

Comment:
The packaging targets from buildtools consume it: https://github.com/dotnet/buildtools/blob/6bc1be1914be31bfa7e11178322cc3f36fe5fb2f/src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets